src/server/common/controller/question-page-controller/question-page-controller.js

```diff
@@ -52,19 +53,54 @@ export class QuestionPageController extends GenericPageController {
       applicationState
     )
 
+    const pageError = req.yar.get(this.errorKey)
+
+    const errorProps = {}
+    if (pageError) {
+      errorProps.pageTitle = `Error: ${this.page.title}`
+      errorProps.errors = pageError.errors
+      errorProps.errorMessages = pageError.errorMessages
+      errorProps.viewModelOptions = {
+        validate: true,
+        question: this.page.question
+      }
+
+      return h.view(this.page.view, {
+        nextPage: req.query.redirect_uri,
+        heading: this.page.heading,
+        answer: pageError
+          ? this.page.Answer.fromState(pageError?.value)
+          : answer,
+        ...errorProps,
+        viewModelOptions: Object.assign(
+          {
+            validate: false,
+            question: this.page.question
+          },
+          errorProps.viewModelOptions
+        ),
```

Comment:
```
viewModelOptions: { 
  validate: false,
  question: this.page.question,
  ...errorProps.viewModelOptions
}
```
?

Reply:
the rules cdp have put in don't like that, as it means a property is overridden (they want the spread first - but that breaks what we are trying to do

so actually we can get rid of the spread altogether now it is in its own seperate branch 

done